APIs/schemas/auth_metadata.json

```diff
@@ -76,5 +68,5 @@
       }
     }
   },
-  "required": ["issuer", "authorization_endpoint", "token_endpoint", "jwks_uri", "registration_endpoint", "revocation_endpoint", "response_types_supported", "code_challenge_methods_supported"]
+  "required": ["issuer", "authorization_endpoint", "token_endpoint", "jwks_uri", "registration_endpoint", "response_types_supported", "code_challenge_methods_supported"]
```

Comment:
@dannymeloy it looks like you removed `"response_types_supported"` above, so doesn't it need removing from the `"required"` list as well?

Reply:
It's actually repeated (its listed further up in the file as well, L40) - wouldn't have realised if it weren't for my IDE throwing a wobbly 😄

Reply:
Ha! OK :-D